seatunnel-e2e/seatunnel-connector-v2-e2e/connector-redis-e2e/src/test/java/org/apache/seatunnel/e2e/connector/redis/RedisIT.java

```diff
@@ -192,4 +192,15 @@ public void testRedis(TestContainer container) throws IOException, InterruptedEx
         jedis.del("key_list");
         Assertions.assertEquals(0, jedis.llen("key_list"));
     }
+
+    @TestTemplate
+    public void testRedisWithExpire(TestContainer container)
+            throws IOException, InterruptedException {
+        Container.ExecResult execResult = container.executeJob("/redis-to-redis-1.conf");
```

Comment:
redis-to-redis-1.conf file name is not clear.
How about use redis-to-redis-expire.conf?

Reply:
done, PTAL.